test/controllers/declaration/SealControllerSpec.scala

```diff
@@ -113,61 +115,40 @@ class SealControllerSpec extends CustomExportsBaseSpec with Generators with Prop
 
       "with valid data and on click of add" in {
 
-        forAll(arbitrary[Seal], listOf[Seal](sealArbitrary.arbitrary)) { (seal, seals) =>
-          withCaching[Seq[Seal]](Some(seals), Seal.formId)
-          val body = Seq(("id", seal.id)) :+ addActionUrlEncoded
-
-          val result = route(app, postRequestFormUrlEncoded(uri, body: _*)).value
-
-          status(result) must be(SEE_OTHER)
-          val updatedSeals = seals :+ seal
-          verify(mockCustomsCacheService)
-            .cache[Seq[Seal]](any(), ArgumentMatchers.eq(Seal.formId), ArgumentMatchers.eq(updatedSeals))(
-              any(),
-              any(),
-              any()
-            )
-        }
+        val body = Seq(("id", "12345")) :+ addActionUrlEncoded
+        val result = route(app, postRequestFormUrlEncoded(uri, body: _*)).value
+        status(result) must be(SEE_OTHER)
+        verify(mockCustomsCacheService)
+          .cache[Seq[Seal]](any(), ArgumentMatchers.eq(Seal.formId), any())(any(), any(), any())
+        theCacheModelUpdated.seals.size mustBe 1
```

Comment:
Maybe a verify what the cache seal contents is?

`theCacheModelUpdated.seals(0) mustBe ...`

Reply:
This has been addressed